test/test_cat.jl

```diff
@@ -33,3 +33,6 @@ rhs = randn(100)
 
 @test_throws LinearOperatorException [LinearOperator(rand(5,5)) ; opEye(3)]
 
+K = [eye(2) opZeros(2,3) ; opZeros(3,2) opEye(3)]
```

Comment:
I think this ends up testing only the new `hcat`, not the new `vcat`. Could you add separate tests for `hcat` and `vcat`?

Reply:
Done. Thanks.